apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -22,27 +22,30 @@ export const MultilingualTextPresenter = ({
     const textItemWithDefaultLanguage =
         items?.find((item) => item.languageCode === defaultLanguageCode) || null;
 
+    const mainText = isNullOrUndefined(textItemWithDefaultLanguage)
```

Comment:
Is there not a unit-tested utility for finding the original text? If there is, use it, if not create it.

Reply:
In fact, we may want a separate "sorting" utility that returns the primary and secondary text item either in an object or array.

Reply:
When I tried the helper, the return of the conditional statement for the (now) `primaryMultilingualTextItem` becomes ambiguous because of the different return type from the helper of `Pick<IMultilingualTextItem, "text" | "languageCode">` rather than just `IMultilingualTextItem`.  

Without creating a more awkward logic to get around this that would make the code less clear, I couldn't find a way to use the helper.

It seems like the "sorting" utility might make this question irrelevant anyway.  Maybe we can punt this to a subsequent PR?